integration/client/computeclass/computeclass_test.go

```diff
@@ -144,6 +162,26 @@ func TestCreatingComputeClasses(t *testing.T) {
 			},
 			fail: true,
 		},
+		// // Raise error to avoid conflicts with
+		// // the "first class" fields for memory and cpu scaling
+		// {
+		// 	name: "invalid-custom-resources-limits",
+		// 	resources: corev1.ResourceRequirements{
+		// 		Limits: corev1.ResourceList{
+		// 			"cpu": resource.MustParse("1"),
+		// 		},
+		// 	},
+		// 	fail: true,
+		// },
+		// {
+		// 	name: "invalid-custom-resources-requests",
+		// 	resources: corev1.ResourceRequirements{
+		// 		Requests: corev1.ResourceList{
+		// 			"memory": resource.MustParse("1"),
+		// 		},
+		// 	},
+		// 	fail: true,
+		// },
```

Comment:
Can this be removed?

Reply:
sure, it was there because I tried to raise errors for the admin when specifying CPU and Memory in the resources field instead that on the Memory and CPU Scale fields. I don't see an easy/useful way to enforce it, since the definition of compute classes does not pass through the acorn client afaiu.

If you have any suggestion I'll give it a try, otherwise I'll go ahead removing this

Reply:
removed, and put into the docs the description of the behavior in case of memory/cpu passed in this field

Reply:
Hello @dciangot. Back from the holiday break. Now that I am properly rested, I better understand what you are saying here. If you would like to add validation for this scenario such that the above (now removed) integration test passes, you would need to add some validation logic [here](https://github.com/acorn-io/runtime/blob/85cfe8c01b57609ae5213adbed04c9e2611e4fb8/pkg/server/registry/apigroups/admin/computeclass/validator.go#L63) and [here](https://github.com/acorn-io/runtime/blob/85cfe8c01b57609ae5213adbed04c9e2611e4fb8/pkg/server/registry/apigroups/admin/computeclass/validator.go#L26).

Adding validation logic in both places would ensure that an admin wouldn't be able to set the memory and cpu on the resources field at the API level.

Would you like to take the time to add this validation now?

Reply:
Sure no problem. I'll add the validation then.